aineko/extras/fastapi/health.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Health endpoint for FastAPI server."""
+
+from fastapi import APIRouter
+
+router = APIRouter()
```

Comment:
We can probably just call this `health_router` and then use 

`from aineko.extras.fastapi.health import health_router `

Reply:
updated here: [42dc84c](https://github.com/aineko-dev/aineko/pull/142/commits/42dc84c8995f32e0b2a788e40c2903ee5a29eb10)